mne_bids/path.py

```diff
@@ -666,8 +666,12 @@ def update(self, *, check=None, **kwargs):
         >>> print(bids_path.basename)
         sub-test_ses-two_task-mytask_channels.tsv
         >>> # Then, one can update this `BIDSPath` object in place
-        >>> bids_path = bids_path.update(acquisition='test', suffix='ieeg',
-        ...                              extension='.vhdr', task=None)
+        >>> bids_path.update(acquisition='test', suffix='ieeg',
+        ...                  extension='.vhdr', task=None)
```

Comment:
this would be the solution, I think

```suggestion
        >>> bids_path.update(acquisition='test', suffix='ieeg',
        ...                  datatype='ieeg',
        ...                  extension='.vhdr', task=None)
```